Source/OpalGraphics/GNUmakefile.postamble

```diff
@@ -0,0 +1,6 @@
+# GNUmakefile.postamble
+
+after-install::
+	cp -r ../../Headers/ImageIO $(shell gnustep-config --variable=GNUSTEP_SYSTEM_HEADERS)/ImageIO
+	cp ../../Headers/CoreGraphics/CGImageSource.h $(shell gnustep-config --variable=GNUSTEP_SYSTEM_HEADERS)/ImageIO
```

Comment:
FWIW, I'd prefer seeing `ImageIO/CGImage{Source,Destination}.h` files which would import the `CoreGraphics/CGImage{Source,Destination}.h` (until the code is properly reorganized so this stuff actually lives in ImageIO).

I'll approve and merge nonetheless.